index.src.html

```diff
@@ -4052,9 +4131,25 @@
     4. Assign `0` to |frame|'s {{VideoFrame/[[coded width]]}},
         {{VideoFrame/[[coded height]]}}, {{VideoFrame/[[visible left]]}},
         {{VideoFrame/[[visible top]]}}, {{VideoFrame/[[visible width]]}},
-        {{VideoFrame/[[visible height]]}}, {{VideoFrame/[[display width]]}},
-        and {{VideoFrame/[[display height]]}}.
-    5. Assign a new {{VideoFrameMetadata}} to |frame|.{{VideoFrame/[[metadata]]}}.
+        {{VideoFrame/[[visible height]]}}, {{VideoFrame/[[rotation]]}},
+        {{VideoFrame/[[display width]]}}, and {{VideoFrame/[[display height]]}}.
+    5. Assign `false` to |frame|'s {{VideoFrame/[[flip]]}}.
+    6. Assign a new {{VideoFrameMetadata}} to |frame|.{{VideoFrame/[[metadata]]}}.
+
+: <dfn for=VideoFrame>Parse Rotation</dfn> (with |rotation|)
+::  1. Let |alignedRotation| be the nearest multiple of `90` to |rotation|.
```

Comment:
What about 225? Let's decide on a direction here. It might be simpler by using simple arithmetic math throughout so it's not ambiguous (since you use the subtraction below).

Reply:
Updated to match css-images draft (https://drafts.csswg.org/css-images/#valdef-image-orientation-angle), which specified rounding towards +infinity.